Xamarin.Forms.Platform.iOS/CollectionView/ItemsViewController.cs

```diff
@@ -28,34 +24,29 @@ public abstract class ItemsViewController : UICollectionViewController
 		public ItemsViewController(ItemsView itemsView, ItemsViewLayout layout) : base(layout)
 		{
 			ItemsView = itemsView;
-			ItemsSource = CreateItemsViewSource();
-
-			UpdateLayout(layout);
+			ItemsViewLayout = layout;
 		}
 
-		public void UpdateLayout(ItemsViewLayout layout)
+		public void UpdateLayout(ItemsViewLayout newLayout)
 		{
-			ItemsViewLayout = layout;
-			ItemsViewLayout.GetPrototype = GetPrototype;
+			// Ignore calls to this method if the new layout is the same as the old one
+			if (CollectionView.CollectionViewLayout == newLayout)
+				return;
 
-			// If we're updating from a previous layout, we should keep any settings for the SelectableItemsViewController around
-			var selectableItemsViewController = Delegator?.SelectableItemsViewController;
```

Comment:
`selectableItemsViewController` was never used. It is now used during layout updates.